src/bluesearch/database/article.py

```diff
@@ -510,14 +511,16 @@ def __str__(self):
         return f'CORD-19 article ID={self.data["paper_id"]}'
 
 
+@deserialize
+@serialize
 @dataclass(frozen=True)
 class Article:
     """Abstraction of a scientific article and its contents."""
 
     title: str
-    authors: Sequence[str]
-    abstract: Sequence[str]
-    section_paragraphs: Sequence[tuple[str, str]]
+    authors: List[str]
+    abstract: List[str]
+    section_paragraphs: List[Tuple[str, str]]
```

Comment:
Is it not possible to switch to `Tuple` here instead of `List` ?

Reply:
That's not what happened.

We changed:
1. for `List` instead of `Sequence`,
2. and for `Tuple` instead of `tuple`.

The (1) is because `pyserde` does not support `Sequence` yet but supports `List`.
The (2) is because `pyserde` does not support the notation `tuple` from Python 3.9+ for earlier Python versions.

Reply:
My question is more, when not changing `Sequence` to `Tuple` ?

Reply:
No typing change is required any more with the new library: `mashumaro`.

I have updated the description of the PR.

Reply:
Following our discussion, I stumble into the fact that `pyserde` doesn't support tuples of variable length.

I went then for another library named `mashumaro`. It is said to be a bit slower. However, it has built-in support for `Sequence`.

Reply:
Thanks a lot for looking at this and changing for `Sequence`. Would you mind adding a comment in `setup.py` next to `mashumaro` (the name is not very explicit) so that we know we use this package for the JSON serialization ? :)

Reply:
Sure. Done in 0fc157f.

Reply:
Sounds good for me! Thanks a lot.